src/Test/nlp.jl

```diff
@@ -337,3 +337,44 @@ const nlptests = Dict("hs071" => hs071_test,
                       )
 
 @moitestset nlp
+
+function test_linear_mcp(model::MOI.ModelLike, config::TestConfig)
```

Comment:
Maybe add the expression of the optimization problem there?

Reply:
And the meaning of the acronym MCP